TradeItIosEmsApi/TradeItInstrumentActionCapability.h

```diff
@@ -1,11 +0,0 @@
-#import "TradeItInstrumentCapability.h"
-
-@protocol TradeItInstrumentActionCapability
-
-@end
-
-@interface TradeItInstrumentActionCapability : TradeItInstrumentCapability
-
-@property (nonatomic, copy, nonnull) NSArray<NSString *> *supportedOrderQuantityTypes;
```

Comment:
This now comes back in `priceType` as well so I just removed this and instead added it as a nullable array to `TradeItInstrumentCapability`